utils/prerelease-id-from-version/lib/prerelease-id-from-version.js

```diff
@@ -0,0 +1,9 @@
+"use strict";
+
+const semver = require("semver");
+
+module.exports = prereleaseIdFromVersion;
+
+function prereleaseIdFromVersion(version) {
+  return (semver.prerelease(version) || []).shift();
+}
```

Comment:
Yes this feels a bit like overkill, but I really couldn't find a better path. This (ridiculously simple) logic is reused in different verticals across the codebase, like `core` and `command`, so it couldn't be dropped in an existing common lib. If you're up for adding a `lib` directory at the top level for little bits like this, I'm happy to drop this commit and add one that does that instead.

Reply:
No worries, this is great!